pkg/cmd/pipelinerun/cancel_test.go

```diff
@@ -369,6 +411,16 @@ func Test_cancel_pipelinerun_v1beta1(t *testing.T) {
 					},
 				},
 			},
+			Status: v1alpha1.PipelineRunStatus{
```

Comment:
Adding status here since the test is for a successful cancel, which means the PipelineRun should be in a running state.